scalding-parquet/src/test/scala/com/twitter/scalding/parquet/tuple/TypedParquetTupleTest.scala

```diff
@@ -0,0 +1,130 @@
+package com.twitter.scalding.parquet.tuple
+
+import java.io.File
+
+import cascading.tuple.{Fields, TupleEntry, Tuple}
+import com.twitter.scalding.typed.TypedPipe
+import com.twitter.scalding.{TupleSetter, TupleConverter, Job, Args, Mode, TypedCsv}
+import scala.io.{Source => ScalaSource}
+
+import org.apache.hadoop
+import org.scalatest.{Matchers, WordSpec}
+
+class TypedParquetTupleTest extends WordSpec with Matchers {
+  "TypedParquetTuple" should {
+
+    "read and write correctly" in {
+
+      val values = Seq(SampleClass("A", 1, 4.0D), SampleClass("B", 2, 3.0D), SampleClass("A", 2, 5.0D))
+
+      //write values to typed parquet sink
+      val tempParquet = createTempDir("parquet_tuple_test_parquet_")
+      val writeJobArgs = buildJobArgs(Array("--output", tempParquet, "--hdfs"))
+      val writeSuccess = new WriteToTypedParquetTupleJob(values, writeJobArgs).run
+      writeSuccess shouldEqual true
+
+      //read from the written parquet tuple and write the values read into a csv file
+      val tempCsv = createTempDir("parquet_tuple_test_csv_")
+      val readJobArgs = buildJobArgs(Array("--input", tempParquet, "--output", tempCsv, "--hdfs"))
+      val readSuccess = new ReadFromTypedParquetTupleJob(readJobArgs).run
+      readSuccess shouldEqual true
+
+      //check data correctness
+      val csv = ScalaSource.fromFile(new java.io.File(tempCsv, "part-00000"))
+      csv.getLines().toList shouldEqual Seq("A,1,4.0", "B,2,3.0", "A,2,5.0")
+
+      //clean temporary files generated by the jobs
+      deletePath(tempCsv)
+      deletePath(tempParquet)
+    }
+  }
+
+  def buildJobArgs(argArray: Array[String]): Args = {
```

Comment:
you should use the platform tests framework for these. None of this code seems too needed/ideal to replicate here?